src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts

```diff
@@ -904,7 +903,10 @@ export class GettingStartedPage extends EditorPane {
 				if (first) {
 					this.currentCategory = first;
 					this.editorInput.selectedCategory = this.currentCategory?.id;
-					this.buildCategorySlide(this.editorInput.selectedCategory);
+					// FIXME: Clarify if this should be in buildCategorySlide.
```

Comment:
@JacksonKearl your input on how this state should be carried over to the UI would be greatly appreciated.